### PR TITLE
Incorrect transaction minFee/size

### LINKED ISSUE #1604: Incorrect transaction minFee/size
### Actual behavior

The transaction `minFee` is lower than the expected value, viz. `size` * `minFeePerByte`.

```
{
  "data": [
    {
      "id": "2206b1e1999c9322118ddd07bf2bb58bef4117246c464e1d84ccef818dd22ec2",
      "moduleCommand": "pos:stake",
      "nonce": "22",
      "fee": "10000000",
      "minFee": "150000",
      "size": 151,
      "sender": {
        "address": "lskzrnxwuno6tdjq7udmkshprexgjbt3sq3jyvyux",
        "publicKey": "3053ee2e4814b1375f25fadd9d798e4a63e65599e8de8cac12c522cc50f9ae14",
        "name": "jsafbbtrxuzoapyruudt"
      },
      "params": {
        "stakes": [
          {
            "validatorAddress": "lskodjmdq7kes4jfez5opunkg66kh2hpm8b6erxcs",
            "amount": "2000000000"
          }
        ]
      },
      "block": {
        "id": "f05ea394c9e808fa5147a922ea2f47e65b2293c97cd27ffc458232fef94135ed",
        "height": 360,
        "timestamp": 1682591860,
        "isFinal": false
      },
      "executionStatus": "success",
      "index": 0
    }
  ],
  "meta": {
    "count": 1,
    "offset": 0,
    "total": 45
  }
}
```

### Expected behavior

- In the above example, the `minFee` must be `151000` or `size` must be `150`.

### Steps to reproduce

- Invoke the `/api/v3/transactions` endpoint

### Which version(s) does this affect? (Environment, OS, etc...)

v0.7.0-beta.0

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #1604 

### How was it solved?

- [x] Verify the current implementation
- [x] Add logic to consider additionalFee if applicable for min fee calculation
- [x] Update files to include all supported config properties:
  - [x] Update `docker/example.env` file
  - [x] Update `ecosystem.config.js` file

### How was it tested?
Local

### CODE REVIEW

File: services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js [hunk 1]
```diff
@@ -45,6 +48,19 @@ const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
 		...tx.params,
 	};
 
+	const filterInitializeUserAccountEvent = events
+		.find(event => event.name === EVENT_NAME_INITIALIZE_USER_ACCOUNT
+			&& event.data.address === tx.params.recipientAddress);
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js:53]: The transaction ID should also be present in the event topics.

File: services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js [hunk 2]
```diff
@@ -45,6 +48,19 @@ const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
...
+
+	if (filterInitializeUserAccountEvent) {
+		const formattedTransaction = await requestConnector('formatTransaction', {
+			transaction: tx,
+			additionalFee: filterInitializeUserAccountEvent.data.initializationFee,
+		});
+
+		tx.minFee = formattedTransaction ? formattedTransaction.minFee : tx.minFee;
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js:61]: Same for other places.
```suggestion
		tx.minFee = formattedTransaction.minFee;
```

File: services/blockchain-indexer/shared/indexer/transactionProcessor/token/transferCrossChain.js [hunk 3]
```diff
@@ -45,6 +48,19 @@ const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
...
+			&& event.data.address === tx.params.recipientAddress);
+
+	if (filterInitializeEscrowAccountEvent) {
+		const formattedTransaction = await requestConnector('formatTransaction', {
+			transaction: tx,
+			additionalFee: filterInitializeEscrowAccountEvent.data.initializationFee,
+		});
+
+		tx.minFee = formattedTransaction ? formattedTransaction.minFee : tx.minFee;
+	}
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/indexer/transactionProcessor/token/transferCrossChain.js:62]: Same as above.

File: services/blockchain-indexer/shared/indexer/transactionProcessor/pos/registerValidator.js [hunk 4]
```diff
@@ -60,6 +66,17 @@ const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
...
 	await validatorsTable.upsert(account, dbTrx);
 	logger.debug(`Indexed validator with address ${account.address}.`);
+
+	const posConstants = await getPosConstants();
+	const formattedTransaction = await requestConnector('formatTransaction', {
+		transaction: tx,
+		additionalFee: posConstants.data.validatorRegistrationFee,
+	});
+
+	tx.minFee = formattedTransaction ? formattedTransaction.minFee : tx.minFee;
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/indexer/transactionProcessor/pos/registerValidator.js:76]: Same as above.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: ecosystem.config.js [hunk 5]
```diff
@@ -46,8 +46,20 @@ module.exports = {
 				HTTP_RATE_LIMIT_WINDOW: 10, // in seconds
 				HTTP_CACHE_CONTROL_DIRECTIVES: 'public, max-age=10',
 				ENABLE_HTTP_CACHE_CONTROL: 'true',
-				HTTP_RATE_LIMIT_ENABLE_X_FORWARDED_FOR: 'false',
+        HTTP_RATE_LIMIT_ENABLE_X_FORWARDED_FOR: 'false',
 				HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES: 0,
```
Comment (vardan10) [inline ecosystem.config.js:50]:
```suggestion
                                HTTP_RATE_LIMIT_ENABLE_X_FORWARDED_FOR: 'false',
				HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES: 0,
```

**Review by vardan10 — APPROVED** [review]

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**